### PR TITLE
I Create a mongoose model for topic 

### PR DESCRIPTION
**Pull request by cardona6**
I Create a mongoose model for topic

### CODE REVIEW

**Comment by rarmatei** [comment]
Thanks for submitting this PR so quickly!!

Before I can merge it in can you please solve task #6 and then look over the comments I made on this PR?

File: models/post.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+//const schema =mongoose.schema;
+
+const schema = new Schema({
+    title: String,
+    contents: String,
+    summary: String
+});
```
Comment (rarmatei) [inline models/post.js]: While title, contents and summary are all good properties for a blog post, they don't really apply for our model `Topic`. 

I'll leave it to you to decide how the model for `Topic` should look (hint: have a look at the table on the main homepage). Let's discuss further once you submit changes for this PR. Thanks!

**Comment by rarmatei** [comment]
#12

**Comment by cardona6** [comment]
this should now be fixed. Please review

**Pull request closed without merging**